data/json/monsters.json

```diff
@@ -411,6 +411,49 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_gas_zombie",
+    "type": "MONSTER",
+    "name": "gasoline zombie",
+    "description": "A huge bloated zombie that appears to have fed upon gasoline, fumes and flames escape from it's mouth and fuel leaks from it's waddling form.",
```

Comment:
"A huge bloated zombie that appears to have fed upon gasoline; fumes and flames escape from its mouth and fuel leaks from its waddling form."